src/components/panels/WebcamPanel.vue

```diff
@@ -100,6 +104,7 @@ import WebcamMixin from '@/components/mixins/webcam'
         'webcam-hlsstreamer': Hlsstreamer,
         'webcam-uv4l-mjpeg': Uv4lMjpeg,
         'webcam-webrtc-camerastreamer': WebrtcCameraStreamer,
+        'webcam-janus': JanusStreamer,
```

Comment:
pls rename it to webcam-webrtc-janus (like the webrtc-camerastreamer example one line above)